RedHat-UX/red-hat-design-system
# fix: pin @patternfly/pfe-tools to version @1.0.0-next.29

fixes #568

## What I did

1. I pinned the version of pfe-tools in our package.json to version 29 for the time being.


## Testing Instructions

1.  checkout this branch `fix/pfe-tools-dep`
2.  run `git clean -fdx` *Make sure you stash your changes before running this command!*
3. run `npm install`
4. run `npm ls @patternfly/pfe-tools`
5. verify that `@patternfly/pfe-tools@1.0.0-next.29` is the only version that should be listed 


![Screen Shot 2022-10-05 at 11 56 24 AM](https://user-images.githubusercontent.com/3428964/194106328-1afda40e-7ccf-4599-b1da-9086895b0f89.png)
